### DISCUSSION TITLE
What's the compatible Laravel versions for the Impersonate package?

### DISCUSSION THREAD

**Discussion by HassanZahirnia** [OctopyID/LaraPersonate#16] [Q&A] [answered]
Hey! 👋😊 I've added your package to my newly launched website: https://laravel-package-ocean.com

There is a small detail missing though, I'm not sure about the compatible Laravel versions for your package? Since it wasn't requiring any Laravel specific dependency in `composer.json`, I couldn't pick it up either.

**Comment by SupianIDz** [comment, marked as answer]
Hi @HassanZahirnia , that's very cool and thanks for adding our package.

For version 3 it supports Laravel 9 and 10 and is tested.

**Reply by HassanZahirnia** [reply]
Thanks for your answer! I've updated the version detail !
